app/javascript/react/components/App.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import RestaurantIndex from "./RestaurantIndex.js"
 import RestaurantShow from "./RestaurantShow.js"
+
 import {Route, Switch, BrowserRouter} from "react-router-dom"
```

Comment:
Line 5 should by next to the import React statement.  Then put an empty line between those two import statements and the ones on 2&3